libs/hw-ledger-key-ring-protocol/package.json

```diff
@@ -24,7 +24,7 @@
     "bip32": "^4.0.0",
     "secp256k1": "^5.0.0",
     "create-hmac": "^1.1.7",
-    "tiny-secp256k1": "1"
+    "tiny-secp256k1": "1.1.7"
```

Comment:
Also the commit message is:
📌 fix(hw-ledger-key-ring-protocol): pin tiny-secp256k1 version to 1.1.6

But the version is `1.1.7`

Reply:
Good catch thanks !